jmix-core/core/src/main/java/io/jmix/core/CoreProperties.java

```diff
@@ -99,6 +99,13 @@ public class CoreProperties {
      */
     boolean roundDecimalValueByFormat;
 
+    /**
+     *
+     * Whether JpqlCondition and Property condition should be skipped by default if property value is null or empty
```

Comment:
Write condition names using a single style: either class names (**JpqlCondition** and **PropertyCondition**) or types using separate words (**JPQL condition** and **property condition**)